src/util/kaldi-holder.cc

```diff
@@ -23,9 +23,11 @@
 
 namespace kaldi {
 
-template<class Real>
-bool ExtractObjectRange(const Matrix<Real> &input, const std::string &range,
-                        Matrix<Real> *output) {
+bool ParseMatrixRangeSpecifier(const std::string &range,
+                         const int rows, const int cols,
```

Comment:
Now that this doesn't have a separate declaration with documentation, it needs some documentatioh.